Doc/library/asyncio.rst

```diff
@@ -4,7 +4,9 @@
 .. module:: asyncio
    :synopsis: Asynchronous I/O.
 
---------------
+**Source code:** :source:`Lib/asyncio/`
+
+---------------------------------------
```

Comment:
This one is also unnecessary, IMO.  I'm fine to keep it if we move it to the footer of the page.

Reply:
Okay, I can move it to the footer then.

Reply:
Done. It now renders as:

![image](https://user-images.githubusercontent.com/44193521/66608770-5a51d180-eb85-11e9-85d8-171d87002d56.png)